setup/system.sh

```diff
@@ -144,6 +144,10 @@ pollinate  -q -r
 
 # Between these two, we really ought to be all set.
 
+echo 'Launching SSH public key creation...'
+
+nohup ssh-keygen -t rsa -b 2048 -a 100 -f /root/.ssh/id_rsa_miab -N '' -q 2>&1 >/dev/null
```

Comment:
nohup ssh-keygen -t rsa -b 2048 -a 100 -f /root/.ssh/id_rsa_miab -N '' -q 2>&1 >/dev/null 2>&1 > /dev/null

This works. 

On my test machine the non detached command completes in under a second though.